.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        
+        Before continuing to make the issue, please make sure there are no similar issues on [the issue tracker](https://github.com/IrisShaders/Iris/issues). If there are, consider contributing your information in there instead!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: 'Also tell us, what did you expect to happen? Please provide some reproduction steps: What did you do to trigger the bug?'
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+      placeholder: You can add images by clicking on the button in the bar that will appear once you click this textbox.
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: dropdown
+    id: edition
+    attributes:
+      label: Edition
+      description: What edition of Iris are you running?
+      options:
```

Comment:
So I'd suggest the following here:

### Iris Version

*String* 

Do you have sodium installed?

*yes | no*